app/src/main/java/ca/bc/gov/bchealth/ui/login/BcscAuthFragment.kt

```diff
@@ -70,7 +75,11 @@ class BcscAuthFragment : BaseSecureFragment(R.layout.fragment_bcsc_auth) {
             when (it) {
                 TermsOfServiceStatus.ACCEPTED -> {
                     if (termsOfServiceViewModel.tosUiState.value.termsOfServiceId != null) {
-                        viewModel.acceptTermsAndService(termsOfServiceViewModel.tosUiState.value.termsOfServiceId!!)
+                        observeCurrentBackStackForAction<TOSStatus>(TOS_STATUS) { tos ->
+                            findNavController().removeActionFromCurrentBackStackEntry<TOSStatus>(TOS_STATUS)
+                            println("TOS, status = $tos")
```

Comment:
I'm assuming this was for debugging, could you remove the `println` ?

Reply:
removed